config/main.toml

```diff
@@ -51,6 +53,26 @@ config_version = 1
         slack_channel = "testing"  # the channel in slack the notifications will be posted
         slack_key_file = "/home/pi/.slack"  # The path the slack key file
 
+    # Step to create HAMMA plot
+    [steps.hamma_plot]
+        _module_path = "hamma_plot"
+        _class_name = "HammaPlot"
+        _is_plugin = true
+        name = "Make HAMMA Plot"
+        save_path = '/home/pi/brokkr/hamma'
+        min_update_time = 30  # seconds
+
+    # Step to rsync HAMMA plot to server
+    [steps.hamma_plot_rsync]
+        _preset = "rsync_server.outputs.plugin"
+        name = "Rsync HAMMA plot"
+        server = "104.131.44.197"  # also in autossh
+        local_path = '/home/pi/brokkr/hamma/'
+        server_path = '/var/www/hamma.dev/public_html/latest'
+        update_time = 120  # seconds
+        include = '*.png'
+        time_key = 'rsync_time_hamma_plot'
```

Comment:
```suggestion
```

No need for this anymore, as it is handled internally as a per-step instance attribute